### ISSUE TITLE
Element Type "HTML" doesn't generate code for Custom Class

### ISSUE THREAD

**Issue by tberneman** [Desertsnowman/Caldera-Forms#500]
I created an element type HTML and gave it a Custom Class but no code is generated to apply the class.
![html-no-class-01](https://cloud.githubusercontent.com/assets/6919675/14109793/5c288e04-f589-11e5-8b46-6c8c0a5f81ea.jpg)
![html-no-class-02](https://cloud.githubusercontent.com/assets/6919675/14109792/5c27dc70-f589-11e5-9d6b-0e9314b778cf.jpg)

**Comment by tberneman** [comment]
Yeah!? The 500th ticket! Do I get a prize! :D

**Comment by Desertsnowman** [comment]
@tberneman congratulations, you get a free copy of Caldera Forms!

**Comment by Desertsnowman** [comment]
also, good point! I'll patch that for the next update

**Comment by Desertsnowman** [comment]
@tberneman that code screen you got there is pointing to content outside of the form. can you confirm that?

**Comment by tberneman** [comment]
It appears that it is generating outside the form but it is part of my form. (see screenshot)

Also, I tried adding a <p> and <span> in my html but it must strip those off because they are not generated in the code. I also dropped in an {ip} as a test and that is not being generated either.

![html-no-class-03](https://cloud.githubusercontent.com/assets/6919675/14111559/d71a154a-f590-11e5-8d3c-fad6ac2f8769.jpg)

**Comment by tberneman** [comment]
Are you serious on the free copy?! That's awesome! How do I collect that? My email is tberneman*at*gmail*dot*com and THANKS SO MUCH! :) :+1:

**Comment by tberneman** [comment]
There may be another issue here as well. I changed "Id" to "ID" and it's not showing that either, even after clearing cache and adding a parameter to the url to force a download from the server. I will dig deeper and let you know if I find anything. The curious thing is it was working fine yesterday so it may be  partially due to a conflict with one of the 2 plugin updates I did this morning.

**Comment by tberneman** [comment]
I don't think there is a bug in your code. I kept having weird problems so I logged off my WordPress Admin and then logged back in and the changes I had made weren't there. I recreated them and saved the form and it is now displaying correctly! I think something was going on in my browser (Firefox) and logging off and on fixed it.

Thank you for your promptness, professionalism, and good humor!

![1609992_1011744978917192_8829638706093574999_n](https://cloud.githubusercontent.com/assets/6919675/14115542/079360a4-f5a1-11e5-8831-4068ecc007b9.jpg)
